OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -1796,6 +1796,27 @@ internal static void UpgradeWeaponRules(ModData modData, int engineVersion, ref
 					}
 				}
 
+				if (engineVersion < 20180120)
```

Comment:
Now that the release is out, please change this to a date after the release (exact day doesn't really matter, 20180219 would be enough).